src/main/java/org/folio/innreach/domain/entity/base/Auditable.java

```diff
@@ -20,21 +20,21 @@
 @ToString
 @MappedSuperclass
 @EntityListeners(AuditingEntityListener.class)
-public class Auditable {
+public class Auditable<T> {
```

Comment:
Made this class parameterized to be able to use a different type for user_id (maybe UUID would be better than String)

Reply:
👍